src/hoc/DragDropHandler.tsx

```diff
@@ -79,7 +60,7 @@ export const DragDropHandler = ({ children, accept, index, id, moveCard, onDropI
   drag(drop(ref))
 
   return (
-    <div ref={ref} style={{ opacity: opacity, margin: '6px 0px' }}>
+    <div ref={ref} style={{ opacity: opacity }}>
```

Comment:
```tsx
style={{ opacity }}
```

Reply:
Changed in commit b5d2a564e35db923a74b0c59da764300ce1a9193